3600/1/gems_stats.rb

```diff
@@ -0,0 +1,19 @@
+# :reek:TooManyInstanceVariables
```

Comment:
You can't disable any errors of reek and rubocop in this and other files, please fix all this errors